### ISSUE TITLE
Django database from google form

### ISSUE THREAD

**Issue by Eragonfr** [nils-van-zuijlen/arbre_competences#37]
How can we fill a Django database from a google form?
Thank you

**Comment by nils-van-zuijlen** [comment]
I don't think you can, but ask ZdS for it.

**Comment by Eragonfr** [comment]
J'avais mit le label Important car c'était important pour permettre un remplissage simple de la bdd, sinon il faudra créer un système pour que les personnes n'étant pas autorisées à ajouter un compte et les n'ayant pas les permissions nécessaires puissent s'ajouter(et si possible depuis n'importe ou sur internet)

**Comment by nils-van-zuijlen** [comment]
Alors demande à ZdS, et renseigne toi sur les API Google Docs, si tu veux utiliser GDocs.
Ou alors, regarde si c'est possible d'avoir une instance de framaforms chez vous / d'utiliser framaforms, puis de fetch régulièrement les données via une API REST / un fichier CSV.

L'avantage de framaforms, c'est que tu peut demander à ce que les personnes valident leur e-mail, et ce peut importe leur provider.

**Comment by Eragonfr** [comment]
Sinon, on peut utiliser l'API de Google pour lire le tableau qui stoque les donnés du formulaire google actuel
[google-api-python-client](https://developers.google.com/sheets/api/quickstart/python)

**Comment by nils-van-zuijlen** [comment]
Je pense qu'il vaut mieux créer une page d'API de notre côté, et l'appeler via l'API forms. Ça évitera les problèmes de duplication liés au tableur.

**Comment by nils-van-zuijlen** [comment]
L'avantage de l'API Forms, c'est que tu peux faire une requète au serveur lors de la soumission du formulaire.
Mais à ce compte là, autant faire un formulaire directement dans le site.

**Comment by Eragonfr** [comment]
Oui, mais le problème est que le site n'est pas accessible depuis tout internet(sauf si on paye le serveur/vps)

**Comment by nils-van-zuijlen** [comment]
Tu connais le DNS et le port forwarding (via NAT/PAT) ? Vous avez des serveurs à l'école, il suffit de partager un des ports 80 avec le reste du monde et de mettre en place un DNS.

**Comment by Eragonfr** [comment]
Mwouais avec une box orange comme celle qu'on a je pense pas que ce soit possible?

**Comment by nils-van-zuijlen** [comment]
Vous avez quoi ?
Demande à Sam pour savoir si c'est possible.

**Comment by Eragonfr** [comment]
Ben en fait on galère déjà à la configurer pour pouvoir utiliser le routeur, mais peut-être que plus tard quand le routeur sera en place... il faudra voir
